src/terra-dev-site/test/application-loading-overlay/LoadingOverlayTest.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import ApplicationLoadingOverlay from '../../../../lib/application-loading-overlay/ApplicationLoadingOverlay';
+import ApplicationLoadingOverlayProvider from '../../../../lib/application-loading-overlay/ApplicationLoadingOverlayProvider';
+
+const ApplicationLoadingOverlayTest = () => {
+  const [showClearOverlay, setShowClearOverlay] = useState(false);
+  const [showLightOverlay, setShowLightOverlay] = useState(false);
+  const [showDarkOverlay, setShowDarkOverlay] = useState(false);
```

Comment:
Does one of these have precedence over the other if multiple are toggled open?

Reply:
The dark does take precedent: https://github.com/cerner/terra-application/pull/1/files#diff-6986970f5522b8cbe14aeb624041cbb7R61